README.md

```diff
@@ -58,13 +58,13 @@ Because it is a directed graph, we can also find predecessor of a given vertex.
 The creation of the graph must be Vertex-typed-constraint. 
 
 For given vertices types: A, B and C 
-For given edge types: F and G
+For given edges types: F and G
```

Comment:
Since types is plural, edge can be singular.

Reply:
done